controllers/infra/configmap/infra_configmap.go

```diff
@@ -49,6 +49,8 @@ func NewWcpClusterConfigMap(wcpClusterConfig WcpClusterConfig) (*corev1.ConfigMa
 		return nil, err
 	}
 
+	fmt.Println(string(bytes))
```

Comment:
Accidental, or you want this here for debugging?

Reply:
On purpose for debugging, but did not mean to leave it. Thanks!